_maps/pubbystation.json

```diff
@@ -0,0 +1,26 @@
+{
+	"version": 1,
+	"map_name": "PubbyStation",
+	"map_path": "map_files/PubbyStation",
+	"map_file": "PubbyStation.dmm",
+	"shuttles": {
+		"emergency": "emergency_pubby",
+		"whiteship": "whiteship_pubby",
+		"ferry": "ferry_fancy",
+		"cargo": "cargo_pubby"
+	},
+	"job_changes": {
```

Comment:
i think this may need to be updated to the new ID system :/

Reply:
The ID changes were done before the map was removed, so the objects are present in the HOP office and functional, i reviewed doors for command and security, where most job changes would be and the *seem* okay, its hard to really judge if its fully functional without running a few test rounds, as for shuttles, ferry has no specific access and the pubby cargo shuttle uses the generic cargo access